src/ngraph_stats.cc

```diff
@@ -54,12 +54,14 @@ void NGraphStats::dump(std::ostream& out) {
           const int pass = static_cast<int>(pass_type);
           std::vector<ngraph::runtime::PerformanceCounter> perf_data =
               backend->get_performance_data(func);
-          if (perf_data.size() > 0) {
+          // TODO: fix this.
+          for (const auto& p : perf_data) {
```

Comment:
I just realized, p is unused, this is going to show a compiler warning

Reply:
i thought we had unused var warning disabled...i guess we can fix this as part of TODO in future.